### DISCUSSION TITLE
Raise issuess of github

### DISCUSSION THREAD

**Discussion by kaushiktalukdar** [community/community#8055] [Projects and Issues]
Hi
could anyone please tell me how do we raise issues if github. I am not aware whether github has any repo to raise bugs for github itself.Github should have it and do let me know if it exists

**Comment by arhadthedev** [comment]
The discussions here actually is a bugtracker for non-crashing issues. However, November 22–28 is Thanksgiving in the US so I suspect most of feedback-related stuff are off-work for a week.
